### ISSUE TITLE
Rolling 'proposed' kernel is not available for ISO distribution release of 'bionic'

### ISSUE THREAD

**Issue by mlopezantequera** [JackHack96/dell-xps-9570-ubuntu-respin#8]
I get this error when trying to run build.sh:

```
~/dell-xps-9570-ubuntu-respin$ ./build.sh ubuntu-18.04-desktop-amd64.iso 
Extracting ISO ...
./isorespin.sh: Rolling 'proposed' kernel is not available for ISO distribution release of 'bionic'.
```

Edit: I'm on 16.04 right now if that helps.

**Comment by JackHack96** [comment]
Mmmh, that's strange. I have an Ubuntu 16.04 server and it builds correctly. Anyway  I've modified the script right now and I'm building it successfully, try it ;)

**Comment by mlopezantequera** [comment]
After your commit, I get the same problem you were getting here: https://github.com/stockmind/dell-xps-9560-ubuntu-respin/issues/6

**Comment by JackHack96** [comment]
Oh fuck. I never solved that problem, it's something about isorespin.sh

**Comment by mlopezantequera** [comment]
I tried using a 18.04 VM, and although the scripts finish, I get a very small .iso (~150mb). This is the terminal output:

```
xubuntu@xubuntu:~/almacen/dell-xps-9570-ubuntu-respin$ sudo ./build.sh ../ubuntu-18.04-desktop-amd64.iso 
Extracting ISO ...
Parallel unsquashfs: Using 1 processor
139912 inodes (156845 blocks) to write

[==|                                                                                                  ]   4171/156845   2%./isorespin.sh: line 2338:  5031 Killed                  sudo unsquashfs mnt/${ISO_SQUASHFS_DIRECTORY}/filesystem.squashfs
Extracting isorespin files ...
Processing bootloader/bootmanager ...
mv: cannot stat 'iso-chroot/run': No such file or directory
cp: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp': No such file or directory
cp: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
chroot: failed to run command 'apt-cache': No such file or directory
chroot: failed to run command 'apt-cache': No such file or directory
mv: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/usr/src/grub-efi-ia32-bin*.deb': No such file or directory
mv: cannot stat 'iso-chroot/usr/src/grub-efi-ia32*.deb': No such file or directory
chroot: failed to run command 'apt-cache': No such file or directory
chroot: failed to run command 'apt-cache': No such file or directory
Adding repositories ...
mv: cannot stat 'iso-chroot/run': No such file or directory
cp: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp': No such file or directory
cp: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
./isorespin.sh: line 2739: iso-chroot/tmp/linuxium.patch: No such file or directory
cat: iso-chroot/tmp/linuxium.patch: No such file or directory
mv: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
Installing packages ...
mv: cannot stat 'iso-chroot/run': No such file or directory
cp: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp': No such file or directory
cp: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
Adding files/directories ...
Running commands ...
mv: cannot stat 'iso-chroot/run': No such file or directory
cp: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp': No such file or directory
cp: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
cat: iso-chroot/root/.command.log: No such file or directory
cat: iso-chroot/root/.command.log: No such file or directory
cat: iso-chroot/root/.command.log: No such file or directory
cat: iso-chroot/root/.command.log: No such file or directory
mv: cannot stat 'iso-chroot/run.linuxium': No such file or directory
mv: cannot stat 'iso-chroot/tmp.linuxium': No such file or directory
Spinning ISO ...
chroot: failed to run command 'dpkg-query': No such file or directory
Parallel mksquashfs: Using 1 processor
Creating 4.0 filesystem on iso-directory-structure/casper/filesystem.squashfs, block size 1048576.
[========================================================================================================|] 2459/2459 100%

Exportable Squashfs 4.0 filesystem, xz compressed, data block size 1048576
	compressed data, compressed metadata, compressed fragments, compressed xattrs
	duplicates are removed
Filesystem size 56678.23 Kbytes (55.35 Mbytes)
	32.44% of uncompressed filesystem size (174712.02 Kbytes)
Inode table size 38738 bytes (37.83 Kbytes)
	21.16% of uncompressed inode table size (183040 bytes)
Directory table size 46212 bytes (45.13 Kbytes)
	40.91% of uncompressed directory table size (112968 bytes)
Number of duplicate files found 1092
Number of inodes 4957
Number of files 3448
Number of fragments 148
Number of symbolic links  934
Number of device nodes 7
Number of fifo nodes 0
Number of socket nodes 0
Number of directories 568
Number of ids (unique uids + gids) 4
Number of uids 1
	root (0)
Number of gids 4
	root (0)
	dip (30)
	shadow (42)
	uuidd (110)
xorriso 1.4.8 : RockRidge filesystem manipulator, libburnia project.

Drive current: -outdev 'stdio:../../linuxium-ubuntu-18.04-desktop-amd64.iso'
Media current: stdio file, overwriteable
Media status : is blank
Media summary: 0 sessions, 0 data blocks, 0 data, 16.3g free
xorriso : WARNING : -volid text problematic as automatic mount point name
xorriso : WARNING : -volid text does not comply to ISO 9660 / ECMA 119 rules
Added to ISO image: directory '/'='/home/xubuntu/almacen/dell-xps-9570-ubuntu-respin/isorespin/iso-directory-structure'
xorriso : UPDATE : 685 files added in 1 seconds
xorriso : UPDATE : 685 files added in 1 seconds
xorriso : NOTE : Copying to System Area: 512 bytes from file '/home/xubuntu/almacen/dell-xps-9570-ubuntu-respin/isorespin/isohdpfx.bin'
libisofs: NOTE : Aligned image size to cylinder size by 336 blocks
xorriso : UPDATE :  13.75% done
xorriso : UPDATE :  74.87% done
ISO image produced: 72704 sectors
Written to medium : 72704 sectors at LBA 0
Writing to 'stdio:../../linuxium-ubuntu-18.04-desktop-amd64.iso' completed successfully.

./isorespin.sh: Respun ISO created as 'linuxium-ubuntu-18.04-desktop-amd64.iso' ... see logfile 'isorespin.log' for details.

```

**Comment by JackHack96** [comment]
It's a bug about isorespin.sh (which isn't mine).
I've built an ISO with the latest kernel available (4.17.9 at time of writing). You can find it [here](https://drive.google.com/open?id=1sjMW3ca-Z6b6AqhrDLvYzirsrgIcUl83)
